bench/config/production_setup.py

```diff
@@ -99,10 +100,24 @@ def disable_production(bench_path='.'):
 
 def service(service_name, service_option):
 	if os.path.basename(which('systemctl') or '') == 'systemctl' and is_running_systemd():
-		exec_cmd(f"sudo systemctl {service_option} {service_name}")
+		status = exec_cmd(f"sudo systemctl {service_option} {service_name}", _raise=False)
+		if service_option == "status":
+			return status == 0
+		if service_option == "reload":
+			if status == 0:
+				exec_cmd(f"sudo systemctl {service_option} {service_name}")
+			else:
+				exec_cmd(f"sudo systemctl start {service_name}")
```

Comment:
Run the same _(reload)_ command again if successful? Did you mean the following or am I missing something?

```suggestion
			if status != 0:
				exec_cmd(f"sudo systemctl start {service_name}")
```

Reply:
Those are not 2x service reloads.
We check service is running. If status == 0 (running), then we reload, else we start service since reloading a non running service would fail.

Reviewing this code made me realize it stopped service to be started stopped. Oops. Made another commit that also improves readability for reload option.